src/validation.h

```diff
@@ -881,6 +878,7 @@ class ChainstateManager
      */
     RecursiveMutex& GetMutex() const LOCK_RETURNED(::cs_main) { return ::cs_main; }
 
+    Options m_options;
```

Comment:
Any reason to remove the `const`?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25905#discussion_r953518739

> Any reason to remove the `const`?

Nope, added const